src/System.Data.Common/src/System/Xml/XmlDataDocument.cs

```diff
@@ -16,8 +16,7 @@ namespace System.Xml
     /// Represents an entire document. An XmlDataDocument can contain XML
     /// data or relational data (DataSet).
     /// </summary>
-    [Obsolete("XmlDataDocument class will be removed in a future release.")]
```

Comment:
Not sure if I should have removed Obsolete

Reply:
Let's leave it in.